modules/account-lib/package.json

```diff
@@ -45,6 +45,7 @@
     "@ethereumjs/common": "^2.4.0",
     "@ethereumjs/tx": "^3.3.0",
     "@hashgraph/sdk": "~2.3.0",
+    "@noble/secp256k1": "git+https://github.com/brandonblack/noble-secp256k1.git#bitgo",
```

Comment:
Should be have this hosted under bitgo org or probably this would be temporary until Paul does the actual release

Reply:
Yeah, if it ends up being much longer before he cuts a release, we should at least cut a release, and probably host under bitgo.